guide/build-config.md

```diff
@@ -69,13 +70,16 @@ milestone: DR1
 brewTagPrefix: fb-1.0-pnc # Optional
 group: Test Product 1.0-all
 releaseStorageUrl: http://testurl.com/registry
+temporaryBuildAlignmentPreference: TEMPORARY
 ```
 The `version` specifies the product version we are building for, and the `milestone` represents the 'step' at which we are in the version. A typical milestone value is DR1, DR2, etc or ER1, ER2 etc. The combination of the version and milestone forms the full release name of the product: e.g 1.0.0.CR1.
 
 The `releaseStorageUrl` is location of the release storage, typically on rcm-guest staging. This is the url to be used in the 'upload to candidates script' when running the `pig release` command. Using `*` in the milestone means that this property may also be used for automatic increment of milestone value. For example `ER*` as milestone, during `pig run`, the command will check the configured release storage and increment the version to next available milestone version. It is possible to configure it either within the `build-config.yaml` or via the CLI using the flag `--releaseStorageUrl=<...>` with the former taking precedence.
 
 The `brewTagPrefix` is an **optional** key that you can use to override the default generated brew tag prefix associated with a product version. The key is used to specify to which Brew tag all the builds should go when the milestone is closed.
 
+The `temporaryBuildAlignmentPreference` is option to specify alignment preference for temporary build to be run with: PERSISTENT or TEMPORARY. This property will override temporary or persistent flags passed in as command line options.
```

Comment:
"is an option"

Reply:
I think it should mention what the default is and what the command line flag is.